optuna/cli.py

```diff
@@ -70,16 +69,37 @@ def get_parser(self, prog_name: str) -> ArgumentParser:
             help="If specified, the creation of the study is skipped "
             "without any error when the study name is duplicated.",
         )
+        parser.add_argument(
+            "--directions",
+            type=str,
+            default=None,
+            choices=("minimize", "maximize"),
+            help="Set directions of optimization to a new study."
+            " Put whitespace between directions. Each direction should be"
+            " either `minimize` or `maximize`.",
+            nargs="+",
+        )
         return parser
 
     def take_action(self, parsed_args: Namespace) -> None:
 
+        if parsed_args.direction is not None:
+            if parsed_args.directions is not None:
+                raise ValueError("Specify only one of `direction` and `directions`.")
+            else:
+                directions = [parsed_args.direction]
+        else:
+            if parsed_args.directions is not None:
+                directions = parsed_args.directions
+            else:
+                directions = ["minimize"]  # default value
```

Comment:
Can you let `create_study` handle this as well, i.e. by passing `direction=parsed_args.direction` and `directions=parsed_args.directions` as they are?

Reply:
It makes sense. Sorry for my insufficient consideration. Dropped it in c14f4b7.